src/app/policies/views/PolicyListView.vue

```diff
@@ -303,6 +301,15 @@ header > h3 {
   margin-top: 0;
   float: left;
 }
+search form {
```

Comment:
this shows up like 4th time, can we make this DRYier?

Reply:
> There is a little CSS repetition here, but I hope to add a XLayout component pretty soon which I hope will cover this at some point.

i.e. not right now, but probably soon yes. There's a PR towards that here

https://github.com/kumahq/kuma-gui/pull/3092

Also, generally I don't mind a little copy/paste for declarative code (and even sometimes imperative). Sometimes its better to copy/paste rather than having lots of small pieces included from elsewhere. Sometimes even the boilerplate required to include the code it longer than the code you are trying to DRY out, or you soon find that you shouldn't have tried DRYing things out in the first place.